util.sh

```diff
@@ -288,9 +288,9 @@ travis_monitor() {
 
 # Check repository for changes, return success(0) if there are changes
 travis_have_fixes() {
-  if ! git diff-index --quiet HEAD -- . ; then  # check for changes in current dir
+  if ! git diff --quiet . ; then  # check for changes in current dir
```

Comment:
We need to use the very same command for checking and (below) for printing diffs.